### ISSUE TITLE
libdav1d

### ISSUE THREAD

**Issue by truexfan81** [mpv-player/mpv-build#239]
please add --enable-libdav1d to scripts/ffmpeg-config

thank you

**Comment by sfan5** [comment]
You can easily do this yourself as described in the README.

**Comment by truexfan81** [comment]
> You can easily do this yourself as described in the README.

Yes i did that on my current build environment, the problem is remembering to do it in the future.  I was not even aware it was required until yesterday when i gave my build of mpv a youtube link that was in AV1 and it wouldn't play it.  To prevent a repeat of that for myself and others i sent a pull request with it added to the script.

**Comment by sfan5** [comment]
I think a good solution would be to enable dav1d by default if it's installed like `scripts/ffmpeg-config` does for either gnutls or openssl.

**Comment by kasper93** [comment]
> I think a good solution would be to enable dav1d by default if it's installed like `scripts/ffmpeg-config` does for either gnutls or openssl.

It's possible, but in this case where do we stop?

**Comment by sfan5** [comment]
The general principle of mpv-build seems to be that the user should enable what he needs, hence my first reply.
But AV1 support is arguably a core features to expect from a video player, so it makes sense to auto-detect.

**Comment by kasper93** [comment]
> But AV1 support is arguably a core feature to expect from a video player, so it makes sense to auto-detect.

Indeed. I would expect ffmpeg to actually provide it by default (autodetect), but it has to be explicitly enabled.
